index.js

```diff
@@ -66,7 +74,14 @@ class Client {
 
       debug('success %s: %j', options.id, body.result || {})
       fn(null, body.result)
-    })
+    }
+
+    if (this.annotateTrace) {
+      var annotated = this.annotateTrace(body.method, requestOptions, callback)
```

Comment:
it is a good practice to use `const` when you're not modifying the reference to a variable